Package.swift

```diff
@@ -7,32 +7,35 @@ let package = Package(
     name: "Starknet",
     platforms: [
         .iOS(SupportedPlatform.IOSVersion.v13),
-        .macOS(SupportedPlatform.MacOSVersion.v12)
+        .macOS(SupportedPlatform.MacOSVersion.v12),
```

Comment:
I believe Ventura is v13 as well, do you plan to update it regularly or will this be set to just oldest macos we want to support?

Reply:
It's the minimum version supported